src/main/java/com/company/M11/Main.java

```diff
@@ -0,0 +1,69 @@
+package com.company.M11;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) {
+//        task1();
+//        task2();
+//        task3();
+//        task4();
+//        task5();
+    }
+
+    private static void task1() {
+        List<String> names = Arrays.asList("John", "Bill", "Anthon", "Alex");
+
+        AtomicInteger index = new AtomicInteger(0);
+        Stream<String> namesStream = names.stream().filter(name -> index.getAndIncrement() % 2 != 0);
+        namesStream.forEachOrdered(name -> System.out.println(index.get() - 1 + ". " + name));
+    }
+
+    private static void task2() {
+        List<String> names = Arrays.asList("John", "Bill", "Anthon", "Alex");
+        Stream<String> namesStream = names.stream().sorted().map(String::toUpperCase);
```

Comment:
Please use this line formatting style when you are working with Streams
```suggestion
        Stream<String> namesStream = names.stream()
        .sorted()
        .map(String::toUpperCase);
```

Try to not create the Stream variable. Finish the working with stream by terminal opereation like forEach(), collect() etc.
Fix here and in all places in the code.

Reply:
fixed